wallet/how-to/use-non-evm-networks/starknet-snap/index.md

```diff
@@ -0,0 +1,174 @@
+---
+sidebar_position: 1
+sidebar_label: Starknet
+description: Learn about how Starknet Snap connects to MetaMask using `get-starknet`.
+tags:
+  - Snaps
+  - non-EVM
```

Comment:
```suggestion
description: Interact with users' Starknet accounts in MetaMask.
```

Syncing with changes I made to the skeleton PR. I think we should leave out doc tags for now, to not over-complicate things.